bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/ItemEntryUtil.scala

```diff
@@ -0,0 +1,32 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.spModel.`type`.{DisplayableSpType, LoggableSpType, SequenceableSpType}
+import edu.gemini.spModel.config2.ItemEntry
+
+/**
+ * Created by jluhrs on 4/27/15.
+ */
+object ItemEntryUtil {
+
+  def showItems(ks: Array[ItemEntry]): String = {
+
+    def width(ks: Array[ItemEntry]): Int = ks match {
+      case Array() => 0
+      case _ => ks.maxBy(_.getKey.toString.length).toString.length
+    }
+
+    def seqValue(o: Object): String = o match {
+      case s: SequenceableSpType => s.sequenceValue()
+      case d: DisplayableSpType => d.displayValue()
+      case l: LoggableSpType => l.logValue()
+      case _ => o.toString
+    }
```

Comment:
I remember having written this but nevertheless I'll comment.  :-) It's easier to read in general if the arrows `=>` line up:

````scala
o match {
  case s: SequenceableSpType => s.sequenceValue()
  case d: DisplayableSpType  => d.displayValue()
  case l: LoggableSpType     => l.logValue()
  case _                     => o.toString
}
````

I know Idea likes to crap on your careful alignment of code so it's a constant struggle.